job-tutorial/cntk/cntk-distributed-example.json

```diff
@@ -0,0 +1,32 @@
+{
+  "jobName": "cntk-distributed-jobguid",
+  "image": "pai.run.cntk",
```

Comment:
>pai.run.cntk [](start = 12, length = 12)

if we don't use pai.build.mpi, why do we build it?

Reply:
user can build mpi into their image (`pai.run.cntk`), or build from mpi base image(`pai.build.mpi`).